api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.6.0"
```

Comment:
if you commit all your changes, and then run hack/update-all.sh, does that work?

I disabled all the git pre-commit hooks in my environment. Not sure if that's a good style though :)